### ISSUE TITLE
comma-dangle enable functions: "never" by default

### ISSUE THREAD

**Issue by aladdin-add** [eslint/eslint#11502]
**What rule do you want to change?**
comma-dangle
**Does this change cause the rule to produce more or fewer warnings?**
more
**How will the change be implemented? (New option, new default behavior, etc.)?**
default behavior: rule option `functions: "never"`
**Please provide some example code that this change will affect:**

```js
foo(0, 1,); // no error
foo(0, 1); // no error
```

**What does the rule currently do for this code?**
no error reported.

**What will the rule do after it's changed?**
```js
foo(0, 1,); // report an error
foo(0, 1); // no error
```

**Are you willing to submit a pull request to implement this change?**
yes.

**Comment by not-an-aardvark** [comment]
This proposal was accepted in [yesterday's TSC meeting](https://github.com/eslint/tsc-meetings/blob/137fbe2be55499cd75f28a008900803078d22dfd/notes/2019/2019-03-14.md#comma-dangle-enable-functions-never-by-default).
